### PR TITLE
Add Morpho and Change OUSD metastrategy allocation calculation

### PR DESCRIPTION
**Pull request by shahthepro**
- Simplifies adding new strategies: In most cases, only `strategies.py` config file would have to be changed
- Renders all strategy card on the UI dynamically
- Updates OUSD metastrategy allocation calculation (shows minuted OUSD and other coin splits) 
- Adds Morpho

Screenshot:
 
image

### CODE REVIEW

File: eagleproject/core/blockchain/addresses.py [hunk 1]
```diff
@@ -178,9 +176,11 @@
...
     OGV_BUYBACK: "OGV BuyBack",
-    REWARDS_SOURCE: "RewardsSource",
+    REWARDS_SOURCE: "RewardsSource"
 }
 
-# Also include metastrategies
-for strat in METASTRATEGIES:
-    CONTRACT_ADDR_TO_NAME[strat["ADDRESS"]] = strat["NAME"]
+# Also include strategies
+for (_, strat) in STRATEGIES.items():
```
Comment (mikeshultz) [inline eagleproject/core/blockchain/addresses.py:183]: FYI, `STRATEGIES.values()`

Comment (shahthepro) [reply eagleproject/core/blockchain/addresses.py:183]: Ah, yes. Will update it

Comment (mikeshultz) [reply eagleproject/core/blockchain/addresses.py:183]: eh, probably not worth fixing specifically, just making you aware if you weren't already.

**Comment by sparrowDom** [comment]
This is pretty awesome thanks @shahthepro . Adding new strategies to analytics will be much easier now

**Pull request merged**